health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/configs/UICustomizations.js

```
@@ -54,6 +55,132 @@ export const UICustomizations = {
+                  header=""
+                  style={{}}
+                /> */}
+              </>
+            )
+            )
+            break;
+          case "CREATION_DATE":
+            let epoch = row?.auditDetails?.createdTime;
+            return Digit.DateUtils.ConvertEpochToDate(epoch);
+            // return row?.auditDetails?.createdTime;
+            break;
```

Biome

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)